ipywidgets/widgets/widget.py

```diff
@@ -483,7 +483,7 @@ def _compare(self, a, b):
         else:
             return a == b
 
-    def _compare_buffers(self, a, b, key):
+    def _buffer_list_equal(self, a, b):
```

Comment:
Actually, I might be over-engineering this one. Other fields in the state other than buffers might also have this issue, so there is no obvious reason to give buffers preferential treatment.